tests/integration/targets/cloudwatchevent_rule/tasks/main.yml

```
@@ -0,0 +1,286 @@
+---
+
+- module_detauls:
+  group/aws:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    security_token: "{{ security_token | default(omit) }}"
+    region: "{{ aws_region }}"
+
+  block:
+    - name: create an ECS cluster
+      ecs_cluster:
+        name: "{{ ecs_cluster_name }}"
+        state: present
+      register: ecs_cluster
+
+    - name: create a VPC to work in
+      ec2_vpc_net:
+        cidr_block: 10.0.0.0/16
+        state: present
+        name: '{{ resource_prefix }}_ecs_cluster'
+        resource_tags:
+          Name: '{{ resource_prefix }}_ecs_cluster'
+      register: setup_vpc
+
+    - name: create subnet
+      ec2_vpc_subnet:
+        az: '{{ ec2_region }}a'
+        tags:
+          Name: '{{ resource_prefix }}_ecs_cluster-subnet-a'
+        vpc_id: '{{ setup_vpc.vpc.id }}'
+        cidr: 10.0.1.0/24
+        state: present
+      register: setup_subnet
+
+    - name: create a security group
+      ec2_group:
+        name: '{{ resource_prefix }}_ecs_cluster-sg'
+        description: 'created by Ansible integration tests'
+        state: present
+        vpc_id: '{{ setup_vpc.vpc.id }}'
+        rules:
+        - ports: 22
+          cidr: 0.0.0.0/0
+      register: setup_sg
+
+    - name: create task definition
+      ecs_taskdefinition:
+        containers: "{{ ecs_fargate_task_containers }}"
+        family: "{{ ecs_task_name }}"
+        launch_type: FARGATE
+        state: present
+      register: ecs_task_definition
+
+    - name: create cloudwatch event rule for integration test
+      register: result
+      cloudwatchevent_rule:
+        name: '{{ resource_prefix }}'
+        description: 'Run Foo'
+        state: enabled
+        schedule_expression: "rate(60 minutes)"
+        targets:
+        - id: run-job-foo
+          arn: "{{ ecs_cluster.arn }}"
+          role_arn: arn:aws:iam::123456789123:role/ecsEventsRole
+          ecs_parameters:
+            launch_type: FARGATE
+            network_configuration:
+              awsvpc_configuration:
+                assign_public_ip: ENABLED
+                security_groups:
+                - "{{ setup_sg.group_id }}"
+                subnets:
+                - "{{ setup_subnet.id }}"
+            task_definition_arn: "{{ ecs_task_definition.taskdefinition.taskDefinitionArn }}:{{ ecs_task_definition.taskdefinition.revision }}"
+            task_count: 1
+
+    - name: check created event rule
+      asserts:
```

```suggestion
      assert:
```
All usages need to be updated.